src/placement/pl_map_common.c

```diff
@@ -320,10 +318,8 @@ determine_valid_spares(struct pool_target *spare_tgt, struct daos_obj_md *md,
 		 * skip this shard.
 		 */
 		if (f_shard->fs_status == PO_COMP_ST_DOWN ||
-		    f_shard->fs_status == PO_COMP_ST_DRAIN) {
+		    f_shard->fs_status == PO_COMP_ST_DRAIN)
 			l_shard->po_rebuilding = 1;
-			f_shard->fs_tgt_id = spare_tgt->ta_comp.co_id;
```

Comment:
Is this 'fs_tgt_id' totally obsolete? I see it's still being used in other places.